internal/config/constants.go

```diff
@@ -68,6 +68,7 @@ const (
 	EnvRootDiskThresholdSize      = "MINIO_ROOTDISK_THRESHOLD_SIZE" // Deprecated Sep 2023
 	EnvBrowserLoginAnimation      = "MINIO_BROWSER_LOGIN_ANIMATION"
 	EnvBrowserSessionDuration     = "MINIO_BROWSER_SESSION_DURATION"
+	EnvMinioStsDuration           = "MINIO_STS_DURATION"
```

Comment:
we need to pass this env variable to console, the same way we are passing MINIO_BROWSER_SESSION_DURATION, and keep suporting both with the intention to deprecate MINIO_BROWSER_SESSION_DURATION. MINIO_STS_DURATION would need to have priority.

Reply:
@adriangitvitz from this discussion you need to move the `if` condition around such that if `MINIO_STS_DURATION` is set it needs to override `MINIO_BROWSER_SESSION_DURATION`